meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr/am64xx-sk/boot.cmd

```diff
@@ -0,0 +1,8 @@
+setenv loadaddr 0x90000000
+setenv devtype mmc
+setenv devnum ${mmcdev}
+run findfdt
+load ${devtype} ${devnum}:2 ${loadaddr} /boot/loader/uEnv.txt
+env import -t ${loadaddr} ${filesize}
+load ${devtype} ${devnum}:2 ${loadaddr} ${kernel_image}
+bootm ${loadaddr}#conf@ti_${fdtfile}
```

Comment:
boot.cmd should really just have enough for loading uEnv.txt (from vfat).

e.g. with freedom:

```
$ cat ./meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr/freedom-u540/boot.cmd
if test ${distro_bootpart} != 1
then
    echo "Boot partition needs to be the first partition"
    exit
fi

fatload ${devtype} ${devnum}:1 $scriptaddr /uEnv.txt
env import -t $scriptaddr $filesize
run bootcmd

$ cat ./meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr/freedom-u540/uEnv.txt.in
bootcmd_otenv=load ${devtype} ${devnum}:3 ${scriptaddr} /boot/loader/uEnv.txt; env import -t ${scriptaddr} ${filesize}
bootcmd_load_f=load ${devtype} ${devnum}:3 ${ramdisk_addr_r} "/boot"${kernel_image}
bootcmd_run=bootm ${ramdisk_addr_r}
bootcmd=run bootcmd_otenv; run bootcmd_load_f; run bootcmd_run
```